src/Controller/CRUDController.php

```diff
@@ -217,7 +217,7 @@ public function deleteAction($id) // NEXT_MAJOR: Remove the unused $id parameter
             return $preResponse;
         }
 
-        if (Request::METHOD_DELETE === $this->getRestMethod()) {
+        if (Request::METHOD_DELETE === $request->getMethod()) {
```

Comment:
**Case 1:**
```yaml
framework:
    http_method_override: false
```
All forms will be generated as `GET|POST` (`Request::getMethod()`). If you want use `DELETE` then you should take it from `_method` parameter.

**Case 2:**
```yaml
framework:
    http_method_override: true
```
Form attribute `method` will be override. If you want use `DELETE` then you can take it from `Request::getMethod()`.

In this case, when people will have this parameter disabled then this if will be treated as allways false.

Reply:
AFAIK, the "_method" request parameter must only receive a special treatment when `Request::$httpMethodParameterOverride` is set to `true`, otherwise it is a common parameter like any other:
* https://symfony.com/doc/current/routing.html#matching-http-methods
>HTML forms only support GET and POST methods. If you’re calling a route with a different method from an HTML form, add a hidden field called _method with the method to use (e.g. <input type="hidden" name="_method" value="PUT"/>). If you create your forms with Symfony Forms this is done automatically for you.
* https://symfony.com/doc/current/forms.html#changing-the-action-and-http-method
>If the form’s method is not GET or POST, but PUT, PATCH or DELETE, Symfony will insert a hidden field with the name _method that stores this method. The form will be submitted in a normal POST request, but Symfony’s routing is capable of detecting the _method parameter and will interpret it as a PUT, PATCH or DELETE request. See the http_method_override option.
* https://symfony.com/doc/current/reference/configuration/framework.html#configuration-framework-http-method-override
>http_method_override¶
>type: boolean default: true
>
>This determines whether the _method request parameter is used as the intended HTTP method on POST requests. If enabled, the Request::enableHttpMethodParameterOverride method gets called automatically. It becomes the service container parameter named kernel.http_method_override.

Reply:
Correct, my bad here.